build.gradle

```diff
@@ -47,6 +47,8 @@ dependencies {
 
     testCompile 'junit:junit:4.12'
     testCompile 'com.winterbe:expekt:0.5.0'
+    testCompile 'org.slf4j:slf4j-log4j12:1.7.25'
```

Comment:
try with org.slf4j:slf4j-simple:1.7.25

http://saltnlight5.blogspot.it/2013/08/how-to-configure-slf4j-with-different.html